pyproject.toml

```diff
@@ -344,6 +347,8 @@ banned-from = [
 typing = "t"
 
 [tool.ruff.lint.flake8-pytest-style]
+parametrize-names-type = "tuple"
+parametrize-values-row-type = "tuple"
```

Comment:
These are the default, so we can probably omit them.

```suggestion
```

- https://docs.astral.sh/ruff/settings/#lint_flake8-pytest-style_parametrize-names-type
- https://docs.astral.sh/ruff/settings/#lint_flake8-pytest-style_parametrize-values-row-type